### PR TITLE
Start on adding deploy docs to README

### PR DESCRIPTION
**Pull request by fractalcat**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -47,8 +44,75 @@ architectural decisions enable this:
...
+
+You need to run four components: the `broker`, plus one or more of each
+of the `writer`, `reader` and `contents` daemons. These can be on the
+same or different hosts, but they must all point to the same broker (if
+you want to make the broker highly-available, use a TCP load-balancer).
+
+The defaults assume your Rados pool is called `vaultaire`, and you're
+connecting with the username `vaultaire`; if this isn't the case,
+override with `-p` and `-u` respectively. In either case, you'll need to
+have the environment variable `CEPH_KEYRING` pointing to the correct
```
Comment (afcowie) [inline README.markdown]: Sure would be nice if we could propagate that in with a command line option, too. I always forget to set it. I supposed we could `setenv()` from inside the program...

File: README.markdown [hunk 2]
```diff
@@ -47,8 +44,75 @@ architectural decisions enable this:
...
+```
+vault -d broker # Start broker in debug mode
+vault -d writer -b localhost # Start writer in debug mode
+vault -d reader -b localhost # Start reader in debug mode
+vault -d contents -b localhost # Start contents daemon in debug mode
+```
+
+# Ecosystem
+
+Vaultaire on its own only provides a binary interface over ZeroMQ. To
```
Comment (afcowie) [inline README.markdown]: @fractalcat This is excellent writing. When you anticipate a [new] users' questions and answer them preemtively it gives the impression that they're in good hands. That's vital for adoption, but even where we aren't per-se trying to convince people to run their own Vaultaires we do want people to feel we know what we're about. This is exactly the sort of things that encourages that.

File: README.markdown (hunk 1 above)
Comment (fractalcat) [inline README.markdown]: Agreed. Created an issue for this (#79).

File: README.markdown (hunk 2 above)
Comment (fractalcat) [inline README.markdown]: Thanks; it helps to have been a new user myself, it just took me a while to get around to translating experience to documentation. :)

**Pull request merged**